packages/core/admin/admin/src/content-manager/components/BlocksInput/tests/BlocksInput.test.tsx

```diff
@@ -90,4 +93,23 @@ describe('BlocksInput', () => {
 
     expect(screen.getByRole('img')).toBeInTheDocument();
   });
+
+  it('should open editor expand portal when clicking on expand button', async () => {
+    const { queryByTestId } = setup({ value: blocksData });
+
+    expect(queryByTestId('blocks-expand')).not.toBeInTheDocument();
```

Comment:
Why do you need the test id? Can't you check for the absence of the close button, or of the absence of the element with role modal?

Reply:
indeed, role modal I thought of but its not very precise to test, checking collapse button is more feasible 👍